examples/LAPACK/foo.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream> // std::cout
+#include <cblas.h>
```

Comment:
I can't cross compile the hunter OpenBlas with mingw-w64. Does it work natively on Windows?

```
[ 75%] Performing build step for 'OpenBLAS'
/bin/sh: 1: ./getarch: not found
make[3]: *** [config.h] Error 127
OpenBLAS: Detecting fortran compiler failed. Cannot compile LAPACK. Only compile BLAS.
gemv.c: In function 'sgemv_':
gemv.c:223:29: error: 'GEMM_MULTITHREAD_THRESHOLD' undeclared (first use in this function)
   if ( 1L * m * n < 2304L * GEMM_MULTITHREAD_THRESHOLD )
                             ^
gemv.c:223:29: note: each undeclared identifier is reported only once for each function it appears in
In file included from ../common.h:747:0,
                 from gemv.c:40:
../common_stackalloc.h:32:46: warning: unused variable 'stack_check' [-Wunused-variable]
 #define STACK_ALLOC_PROTECT_SET volatile int stack_check = 0x7fc01234;
                                              ^
../common_stackalloc.h:56:3: note: in expansion of macro 'STACK_ALLOC_PROTECT_SET'
   STACK_ALLOC_PROTECT_SET                                                  \
   ^
gemv.c:219:3: note: in expansion of macro 'STACK_ALLOC'
   STACK_ALLOC(buffer_size, FLOAT, buffer);
   ^
Makefile:841: recipe for target 'sgemv.obj' failed
make[4]: *** [sgemv.obj] Error 1
Makefile:126: recipe for target 'libs' failed
make[3]: *** [libs] Error 1
```